src/ApiProblem.php

```diff
@@ -135,7 +135,7 @@ class ApiProblem implements \ArrayAccess, \JsonSerializable
      */
     public static function fromJson(string $json) : self
     {
-        if (empty($json)) {
+        if ($json === '') {
```

Comment:
```suggestion
        if (trim($json) === '') {
```

An all-whitespace string wouldn't be valid JSON either, I believe.

Reply:
Indeed you are correct. I'll update this code